src/sync-client.js

```diff
@@ -1289,5 +1292,6 @@ module.exports = {
   setCloudHandler: self.setCloudHandler,
   doCloudCall: self.doCloudCall,
   setStorageAdapter: self.setStorageAdapter,
-  setHashMethod: self.setHashMethod
+  setHashMethod: self.setHashMethod,
+  isOnline : self.isOnline
```

Comment:
I was thinking about it, but in typescript it;s not possible to override method like in javascript. Proper aproach here will be to use strategy pattern and add another method `setNetworkStatusHandler` that will accept actual method. Going this way will enable typescript. It will be the same aproach as `setHashMethod` etc. WDYT?